src/shipit_bot_uplift/shipit_bot_uplift/report.py

```diff
@@ -0,0 +1,53 @@
+from shipit_bot_uplift import log
+import taskcluster
+
+
+logger = log.get_logger('shipit_bot')
+
+
+class Report(object):
+    """
+    Build a report during bot execution
+    and send it at the end through TC emails
+    """
+
+    def __init__(self, tc_options, emails):
```

Comment:
It makes it very hard to read when you used short names of variables, i would rename - in the future - ``tc_options`` with ``taskcluster_options``.